src/routes/NonAuthLayoutRouter/NonAuthLayoutRouter.js

```diff
@@ -49,6 +50,10 @@ export default function NonAuthLayoutRouter (props) {
 
   return (
     <Div100vh styleName='nonAuthContainer'>
+      <Helmet>
+        <title>Hylo</title>
+        <meta name='description' content='Login/signup to see more' />
```

Comment:
We should have a generic description of Hylo here. e.g. Prosocial Coordination for a Thriving Planet

Reply:
ok, I can add that